test/testSamples/MaterialX/DemoQuads.usda

```diff
@@ -581,7 +581,7 @@ def Scope "mtl"
         def Shader "normalmap1"
         {
             uniform token info:id = "ND_normalmap"
-            float3 inputs:normal.connect = </mtl/standard_surface1/image1.outputs:out>
+            float3 inputs:in.connect = </mtl/standard_surface1/image1.outputs:out>
```

Comment:
Incorrect connection fixed.

Reply:
[Question] Does this .usda file store in some way which MaterialX version it usus? "normal" would be correct in 1.38, right?

I think you mentioned at some point that all USD files using MaterialX 1.39 will explicitly specify that. If nothing is specified, 1.38 is assumed. Is that correct?

If so, do we need an additional change in this file to specify that it's now using 1.39? Or should the name remain as "normal" and tests use the upgrade path to convert it so 1.39 before running?

Reply:
Data without a version is to be assumed 1.38. There is a new [MaterialXConfigAPI](https://github.com/PixarAnimationStudios/OpenUSD/blob/dev/pxr/usd/usdMtlx/schema.usda) to be used to identify MaterialX versions. You can see it in use in [this file](https://github.com/PixarAnimationStudios/OpenUSD/blob/d2f309cb30c7b811b34dd5c9b643791472092af6/pxr/usd/usdMtlx/testenv/testUsdMtlxFileFormat.testenv/baseline/usd_preview_surface_gold.usda#L8).

Reply:
The version info is propagated thru Hydra, so the material received by the VP2 render delegate for this data is known to be 1.38. The code in hdMtlx will convert the HdMaterialNetwork by first [setting the version ](https://github.com/PixarAnimationStudios/OpenUSD/blob/d2f309cb30c7b811b34dd5c9b643791472092af6/pxr/imaging/hdMtlx/hdMtlx.cpp#L709)it received, than [upgrading it](https://github.com/PixarAnimationStudios/OpenUSD/blob/d2f309cb30c7b811b34dd5c9b643791472092af6/pxr/imaging/hdMtlx/hdMtlx.cpp#L742) to the current one before returning the MaterialX document.

Reply:
Got it! That's very good to know, thanks!

I actually got confused here. I thought the input got renamed in the new MaterialX version, but that connection was genuinely wrong independent of the MaterialX version.